frontend/src/app/starters/InterestsStarter.tsx

```diff
@@ -9,7 +9,7 @@ interface InterestsStarterProps {
 // TODO: Implement your component
 // eslint-disable-next-line no-empty-pattern
 const InterestsStarterInner = ({}: InterestsStarterProps) => {
-    return <div />
+    return <p>Liliana Ortiz was here!</p>
```

Comment:
Not sure why this is showing up in your diff.

Try to update your branch with the latest code from the main branch again.

```bash
# Go to the main branch
git checkout main
# Pull the latest code
git pull
# Switch back to your branch, replace with your branch name
git checkout YOUR_BRANCH_NAME
# Merge the latest code into your branch
git merge main
# Resolve any conflicts and then push
git push
```